### ISSUE TITLE
レイアウトのロックが復元になってる。

### ISSUE THREAD

**Issue by KimidoriBoushi** [gre4bee/ElectronicObserver#121]
レイアウト変更してロックしようとすると、変更前の状態になります。
ウィンドウ位置も前に保存したのであろう場所に移動します。

保存してからロックすると、その変更状態で固定できます。

多分レイアウトロックと復元がイコールになっているのではないかと。

**Comment by myangelkamikaze** [comment]
Yeah we need to reload the layout whenever changing the grid splitter size. Locking the layout works by setting grid splitter size to 0. We're thinking about saving first before reloading so you'd keep your current layout.
